backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalService.java

```diff
@@ -103,13 +114,15 @@ public String getVersion() {
 
     @GET
     @Path("/")
+    @RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_USER)
```

Comment:
Same question than in `InternalPermissionService` for multiple methods in this class: should we do this?
```suggestion
    @RolesAllowed({ ConsoleIdentityProvider.RBAC_INTERNAL_ADMIN, ConsoleIdentityProvider.RBAC_INTERNAL_USER })
```

Reply:
I think the `ConsoleIdentityProvider.RBAC_INTERNAL_USER` permission is enough. It's given to all the users (even the admin)